internal/k8s/reconciler.go

```diff
@@ -25,20 +25,45 @@ import (
 	"github.com/vdaas/vald/internal/errors"
 	"github.com/vdaas/vald/internal/net"
 	"github.com/vdaas/vald/internal/safety"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/builder"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	cli "sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+// TODO delete <<<---
+
+type (
+	Manager        = manager.Manager
+	OwnerReference = v1.OwnerReference
+)
+
+type (
+	Object             = cli.Object
+	ObjectKey          = cli.ObjectKey
+	DeleteAllOfOptions = cli.DeleteAllOfOptions
+	ListOptions        = cli.ListOptions
+	MatchingLabels     = cli.MatchingLabels
+	InNamespace        = cli.InNamespace
+)
+
+// TODO delete --->>>
+
 type Controller interface {
 	Start(ctx context.Context) (<-chan error, error)
+	// TODO delete <<<---
+	GetManager() Manager
+	// TODO delete --->>>
 }
```

Comment:
📝 It will be adjusted with the PR of [continuous benchmark tool](https://github.com/vdaas/vald/pull/1951)